### PR TITLE
Add option to ignore scilla checker for outdated contract

### PR DESCRIPTION
**Pull request by kaikawaliu**
## Description

1) Add option to ignore scilla checker for migration in case some outdated contract affecting the testnet
2) Remove extra quotes during migration to make states inconsistent
3) Enable empty map key in states json api

## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: src/cmd/data_migrate.cpp [hunk 1]
```diff
@@ -25,30 +29,72 @@
...
+#define ERROR_UNHANDLED_EXCEPTION -2
+
 using namespace std;
-int main() {
+namespace po = boost::program_options;
+
+int main(int argc, const char* argv[]) {
   PairOfKey key;  // Dummy to initate mediator
   Peer peer;
+  string ignore_checker_str;
```
Comment (ShengguangXiao) [inline src/cmd/data_migrate.cpp]: ignore_checker_str default should initialize to "false"

Comment (kaikawaliu) [reply src/cmd/data_migrate.cpp]: The bool will be passed by the comparison with the string to "true". so it's unnecessary to initialize.

Comment (ShengguangXiao) [reply src/cmd/data_migrate.cpp]: It is not a bool type, it is a string type

Comment (ShengguangXiao) [reply src/cmd/data_migrate.cpp]: Ok, i find if it is empty, ignore_checker also becomes false

**Pull request merged**